subsys/net/lib/aws_iot/src/aws_iot.c

```diff
@@ -0,0 +1,679 @@
+#include <net/mqtt.h>
+#include <net/socket.h>
+#include <net/cloud.h>
+#include <net/cloud_backend.h>
+#include <at_cmd.h>
```

Comment:
Wondering if we can have a Kconfig that includes this if the user wants to have IMEI as ID